### ISSUE TITLE
System.Text.Formatting.Tests assume US UICulture

### ISSUE THREAD

**Issue by codespare** [dotnet/corefxlab#292]
With:
```
Culture:   en-US
UICulture: en-AU
```
```
  xUnit.net console test runner (32-bit .NET Core)
  Copyright (C) 2014 Outercurve Foundation.

  Discovering: System.Text.Formatting.Tests
  Discovered:  System.Text.Formatting.Tests
  Starting:    System.Text.Formatting.Tests
     System.Text.Formatting.Tests.SystemTextFormattingTests.FormatDateTimeOffsetG [FAIL]
        Assert.Equal() Failure
                  Γåô (pos 0)
        Expected: 3/10/2015 5:31:59 PM
        Actual:   10/3/2015 5:31:59 PM
                  Γåæ (pos 0)
        Stack Trace:
           .\corefxlab\src\System.Text.Formatting\tests\PrimitiveFormattingTests-Time.cs(42,0): at System.Text.Formatting.Tests.SystemTextFormattingTests.FormatDateTimeOffsetG()
     System.Text.Formatting.Tests.SystemTextFormattingTests.FormatDateTimeG [FAIL]
        Assert.Equal() Failure
                  Γåô (pos 0)
        Expected: 3/10/2015 5:31:59 PM
        Actual:   10/3/2015 5:31:59 PM
                  Γåæ (pos 0)
        Stack Trace:
           .\corefxlab\src\System.Text.Formatting\tests\PrimitiveFormattingTests-Time.cs(75,0): at System.Text.Formatting.Tests.SystemTextFormattingTests.FormatDateTimeG()
  InvariantFormatIntDec : Elapsed 853ms
  InvariantFormatIntDecClr : Elapsed 251ms
  InvariantFormatIntHex : Elapsed 513ms
  InvariantFormatIntHexClr : Elapsed 239ms
  InvariantFormatStruct : Elapsed 1131ms
  FormatGuid : Elapsed 1009ms
  InvariantFormatStructClr : Elapsed 432ms
  CustomCultureFormat : Elapsed 1037ms
  CustomCultureFormatClr : Elapsed 176ms
  EncodeStringToUtf8 : Elapsed 514ms
  EncodeStringToUtf8Clr : Elapsed 77ms
     System.Text.Formatting.Tests.CompositeFormattingTests.CompositeFormattingBasics [FAIL]
        Assert.Equal() Failure
                  Γåô (pos 0)
        Expected: 3/10/2015 5:32:05 PM - Error 404. File in┬╖┬╖┬╖
        Actual:   10/3/2015 5:32:05 PM - Error 404. File in┬╖┬╖┬╖
                  Γåæ (pos 0)
        Stack Trace:
           .\corefxlab\src\System.Text.Formatting\tests\CompositeFormattingTests.cs(18,0): at System.Text.Formatting.Tests.CompositeFormattingTests.CompositeFormattingBasics()
  Finished:    System.Text.Formatting.Tests

  === TEST EXECUTION SUMMARY ===
     System.Text.Formatting.Tests  Total: 83, Errors: 0, Failed: 3, Skipped: 0, Time: 6.426s
```

**Comment by alexanderkozlenko** [comment]
It seems that the issue is already fixed with #497 and #506

**Comment by KrzysztofCwalina** [comment]
@codespare, should I close?
